### PR TITLE
fix very important typos

### PR DESCRIPTION
**Pull request by mustafa-archived**
I fixed a few typos in the repo, pls merge

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -20,11 +20,11 @@ jobs:
     - uses: actions/checkout@v3
       with:
         submodules: recursive
-    - name: Setup .NET
+    - name: Setup DotNET
```
Comment (jackcaver) [inline .github/workflows/build.yml:23]: Setup .NET is the default name that GitHub Actions gives to this step, what's wrong with it?

**Comment by mustafa-archived** [comment]
:serios:

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-name: Build
+name: Build API
```
Comment (jackcaver) [inline .github/workflows/build.yml:1]: What's wrong with "Build"? API is the only thing that is being built in this repo

Comment (mustafa-archived) [reply .github/workflows/build.yml:1]: Fair enough

**Comment by jackcaver** [comment]
i don't think that there is something wrong with this file, so i'm closing this PR

**Comment by mustafa-archived** [comment]
Aight lol

**Pull request closed without merging**